File: 1036.py
A, B, C = [float(x) for x in input().split()]

DT = (B * B) - (4 * A * C);

if A != 0 and DT > 0:
    R1 = (-B + DT**0.5)/(2 * A);
    R2 = (-B - DT**0.5)/(2 * A);
    print('R1 = {:.5f}'.format(R1))
    print('R2 = {:.5f}'.format(R2))
else: print('Impossivel calcular')

File: 1-Iniciante/1036.cpp
#include <iostream>
#include <iomanip>
#include <math.h>

using namespace std;

int main(void)
{
    double A, B, C, R1, R2, DT;

    cin >> A >> B >> C;

    DT = (B * B) - (4.0 * A * C);
    R1 = (-B + sqrt(DT))/(2.0 * A);
    R2 = (-B - sqrt(DT))/(2.0 * A);

    if (A != 0.0 && DT > 0) {
        cout << "R1 = " << fixed << setprecision(5) << R1 << endl;
        cout << "R2 = " << fixed << setprecision(5) << R2 << endl;
    }
    else cout << "Impossivel calcular" << endl;

    return 0;
}
